docs/eventing/broker-trigger.md

```diff
@@ -124,16 +124,19 @@ required once per namespace. These instructions will use the `default`
 namespace, but you can replace it with any namespace you want to install a
 `Broker` into.
 
-Create the `ServiceAccount`.
+Create the `ServiceAccount` objects.
 
 ```shell
 kubectl -n default create serviceaccount eventing-broker-ingress
 kubectl -n default create serviceaccount eventing-broker-filter
 ```
 
-Then give it the needed RBAC permissions:
+Then give them the needed RBAC permissions:
 
 ```shell
+kubectl -n default create rolebinding eventing-broker-ingress \
```

Comment:
137-139 are exactly the same as 140-142, added by mistake?

Reply:
Good catch! This command was added by mistake.